tests/output/ConsoleInteractionTest.py

```diff
@@ -151,23 +151,27 @@ def tearDown(self):
     def test_require_settings(self):
         self.assertRaises(TypeError, acquire_settings, self.log_printer, 0)
 
+        curr_section = Section("")
```

Comment:
Yep, another assert statement in `test_fail_acquire_settings` will do. No need for a new function :+1: